go/batch-submitter/config.go

```diff
@@ -74,14 +74,18 @@ type Config struct {
 	// by the batch submitter.
 	MaxL1TxSize uint64
 
+	// MinStateRootElements is the minimum number of state root elements that
+	// can be submitted in single proposer batch.
+	MinStateRootElements uint64
+
+	// MaxStateRootElements is the maximum number of state root elements that
+	// can be submitted in single proposer batch.
+	MaxStateRootElements uint64
+
 	// MaxTxBatchCount is the maximum number of L2 transactions that can ever be
 	// in a batch.
 	MaxTxBatchCount uint64
 
-	// MaxStateBatchCount is the maximum number of L2 state roots that can ever
-	// be in a batch.
-	MaxStateBatchCount uint64
```

Comment:
If I understand correctly, this was unused previously

Reply:
yes it was, the member existed but it wasn't read/written anywhere